fd-lock change breaks build on windows

56b925c271b56da77e7a78a64470e239bc1daa09 breaks the build on my Windows machine.

```
   Compiling dirs-next v2.0.0
error[E0308]: mismatched types
  --> C:\Users\...\.cargo\registry\src\github.com-1ecc6299db9ec823\fd-lock-2.0.0\src\windows.rs:39:33
   |
39 |         if unsafe { !UnlockFile(handle, 0, 0, 1, 0) } == 0 {
   |                                 ^^^^^^ expected enum `winapi::ctypes::c_void`, found enum `std::ffi::c_void`
   |
   = note: expected raw pointer `*mut winapi::ctypes::c_void`
              found raw pointer `*mut std::ffi::c_void`

error[E0308]: mismatched types
  --> C:\Users\...\.cargo\registry\src\github.com-1ecc6299db9ec823\fd-lock-2.0.0\src\windows.rs:73:35
   |
73 |         match unsafe { LockFileEx(handle, flags, 0, 1, 0, overlapped.raw()) } {
   |                                   ^^^^^^ expected enum `winapi::ctypes::c_void`, found enum `std::ffi::c_void`
   |
   = note: expected raw pointer `*mut winapi::ctypes::c_void`
              found raw pointer `*mut std::ffi::c_void`

error[E0308]: mismatched types
  --> C:\Users\...\.cargo\registry\src\github.com-1ecc6299db9ec823\fd-lock-2.0.0\src\windows.rs:90:33
   |
90 |         match unsafe { LockFile(handle, 0, 0, 1, 0) } {
   |                                 ^^^^^^ expected enum `winapi::ctypes::c_void`, found enum `std::ffi::c_void`
   |
   = note: expected raw pointer `*mut winapi::ctypes::c_void`
              found raw pointer `*mut std::ffi::c_void`
```

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -41,7 +41,7 @@ utf8parse = "0.2"
 skim = { version = "0.9", optional = true }
 
 [target.'cfg(windows)'.dependencies]
-winapi = { version = "0.3", features = ["consoleapi", "handleapi", "minwindef", "processenv", "winbase", "wincon", "winuser"] }
+winapi = { version = "0.3", features = ["consoleapi", "handleapi", "minwindef", "processenv", "std", "winbase", "wincon", "winuser"] }
 scopeguard = "1.1"
 
 [dev-dependencies]

